src/main/java/com/github/scuwr/snitchvisualizer/handlers/ReflectionHandler.java

```diff
@@ -0,0 +1,26 @@
+package com.github.scuwr.snitchvisualizer.handlers;
+
+import java.lang.reflect.Field;
+
+import net.minecraft.client.Minecraft;
+
+/**
+ * Because rendering shit in mc now requires reflection!
+ * 
+ * @author TealNerd
+ *
+ */
+public class ReflectionHandler {
```

Comment:
I totally believe you but this is kind of a crazy move against modding on Minecraft's part. Ugh.